rust/feature-flags/src/api.rs

```diff
@@ -98,7 +127,7 @@ impl IntoResponse for FlagError {
                 (StatusCode::BAD_REQUEST, "The distinct_id field is missing from the request. Please include a valid identifier.".to_string())
             }
             FlagError::NoTokenError => {
-                (StatusCode::UNAUTHORIZED, "No API key provided. Please include a valid API key in your request.".to_string())
+                (StatusCode::UNAUTHORIZED, "No API token provided. Please include a valid API token in your request.".to_string())
```

Comment:
not sure if we should call it a key or a token, lol.  We'll pick one on which to standardize.